.github/workflows/pkgdown.yaml

```diff
@@ -33,7 +33,7 @@ jobs:
         shell: Rscript {0}
 
       - name: Configure pages
-        uses: actions/configure-pages@v3
+        uses: actions/configure-pages@v5
```

Comment:
Required due to the [v3 `upload-artifact` deprecation](https://github.blog/changelog/2024-04-16-deprecation-notice-v3-of-the-artifact-actions/).